### ISSUE TITLE
PWA

### ISSUE THREAD

**Issue by GihanSoft** [troyeguo/koodo-reader#579]
### I've read Koodo Reader's roadmap, there is no similar feature.

- [X] Yes

### This issue only requests one feature.

- [X] Yes

### Brief Introduction

Please make web version a PWA

### Details

Users don't need a chrome browser in their PC with koodo-reader (nor node.js). they have a browser. So if you make web version a PWA users can just install it and enjoy it light weight and easy.
Best scenario is it to became a full flag app but it can be a light version if some feature need native access.
(browsers give lots of native access with a permission ask, like file system, local fonts, even usb and serial ports)

**Comment by Jaifroid** [comment]
Yes, please make Koodoo Reader into a PWA using an offline-first Service Worker strategy. Also make it installable in Chromium browsers. You're already 95% of the way there, since you have local storage of ebooks and it all runs in the browser without actually needing the Electron framework. You could even use the File System Access API in order to allow users to link to their ebooks with the PWA rather than importing them into IndexedDB (you just store the file handle).
